setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+package_name = 'mock_ros2_project'
+
+setup(
+    name=package_name,
+    version='0.0.1',
+    packages=[package_name],
```

Comment:
Using a hard-coded list (`packages=[package_name]`) will not pull in sub-packages (e.g. utilities or message helpers) should they be added later. Switching to `packages=find_packages(include=[package_name, f"{package_name}.*"])` will make the package discovery future-proof.

```suggestion
    packages=find_packages(include=[package_name, f"{package_name}.*"]),
```